src/sentry/static/sentry/app/components/tooltip.tsx

```diff
@@ -96,21 +97,18 @@ class Tooltip extends React.Component<Props, State> {
     containerDisplayMode: 'inline-block',
   };
 
-  state = {
+  state: State = {
     isOpen: false,
     usesGlobalPortal: true,
   };
 
   componentDidMount() {
-    if (!window.__tooltips) {
-      window.__tooltips = [];
-    }
-    window.__tooltips.push(this);
+    TooltipStore.addTooltip(this);
```

Comment:
I would consider dynamically importing `app/stores/tooltipStore/` based on `app/constants/IS_CI` so that it's not included in production builds

Reply:
You mean dynamic import via React.lazy? Does that work for non-components / typescript?

Reply:
Not with `React.lazy`, you only need `import()`

Reply:
Ah yeah that was the typescript part of that question, sorry. It comes back with `typeof <filename>` which doesn't resolve. I was considering just casting it to any.